typings/pinejs-client-core.d.ts

```diff
@@ -95,14 +102,14 @@ type ExpandedProperty<
 					: never;
 
 export type ExpandResultObject<T, Props extends keyof T> = {
-	[P in Props]: ExpandedProperty<T, P, object>;
+	[P in Props]-?: ExpandedProperty<T, P, object>;
 };
 
 type ExpandResourceExpandObject<
 	T,
 	TResourceExpand extends ResourceExpand<T>,
 > = {
-	[P in keyof TResourceExpand]: ExpandedProperty<
+	[P in keyof TResourceExpand]-?: ExpandedProperty<
```

Comment:
Now that the reverse navigation properties are always optional this removes their optionality for when they are expanded